models/facial_expression_recognition/quantize-inc.py

```diff
@@ -0,0 +1,125 @@
+import os
```

Comment:
I think this file is basically the same with the one in tools/quantize except the different default scale used in `Dataset`. Could you update `tools/quantize/quantize-inc.py` with the changes in this file and remove this file?

Others look good to me. I can help you rebase commits so that we can merge this pull request.